cmd/mr_discussion.go

```diff
@@ -49,6 +47,17 @@ var mrCreateDiscussionCmd = &cobra.Command{
 			log.Fatal(err)
 		}
 
+		mr, err := lab.MRGet(rn, int(mrNum))
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		state := map[string]string{
+			"opened": "OPEN",
+			"closed": "CLOSED",
+			"merged": "MERGED",
+		}[mr.State]
+
```

Comment:
Why not calling `noteGetState()` here with `isMR = true` instead of replicating the code?

Reply:
#oops.  I thought I had done that.  Fixed in new version.